### PR TITLE
[FrameworkBundle] Fix class_exists() checks in PhpArrayAdapter-related cache warmers

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Follow up of #21102

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AnnotationsCacheWarmer.php [hunk 1]
```diff
@@ -36,8 +36,8 @@ class AnnotationsCacheWarmer implements CacheWarmerInterface
 
     /**
      * @param Reader                 $annotationReader
-     * @param string                 $phpArrayFile     The PHP file where annotations are cached.
-     * @param CacheItemPoolInterface $fallbackPool     The pool where runtime-discovered annotations are cached.
+     * @param string                 $phpArrayFile     the PHP file where annotations are cached
+     * @param CacheItemPoolInterface $fallbackPool     the pool where runtime-discovered annotations are cached
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AnnotationsCacheWarmer.php]: Maj should be reverted, dots can be removed indeed

File: src/Symfony/Bundle/FrameworkBundle/CacheWarmer/SerializerCacheWarmer.php [hunk 2]
```diff
@@ -36,9 +37,9 @@ class SerializerCacheWarmer implements CacheWarmerInterface
     private $fallbackPool;
 
     /**
-     * @param LoaderInterface[]      $loaders      The serializer metadata loaders.
-     * @param string                 $phpArrayFile The PHP file where metadata are cached.
-     * @param CacheItemPoolInterface $fallbackPool The pool where runtime-discovered metadata are cached.
+     * @param LoaderInterface[]      $loaders      the serializer metadata loaders
+     * @param string                 $phpArrayFile the PHP file where metadata are cached
+     * @param CacheItemPoolInterface $fallbackPool the pool where runtime-discovered metadata are cached
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/CacheWarmer/SerializerCacheWarmer.php]: same

File: src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AnnotationsCacheWarmer.php (hunk 1 above)
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AnnotationsCacheWarmer.php]: reverted

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

File: src/Symfony/Bundle/FrameworkBundle/composer.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/cache": "~3.2",
+        "symfony/cache": "~3.2.2|~3.3",
```
Comment (sstok) [inline src/Symfony/Bundle/FrameworkBundle/composer.json]: Why not `^3.2.2`?

**Pull request merged**